### DISCUSSION TITLE
Show blame when viewing file

### DISCUSSION THREAD

**Discussion by brian-lutz-informed** [community/community#4008] [Mobile] [answered]
It'd be nice if when viewing a file in the mobile app a user could show blame like in the web app. Thank you!

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to view git blame internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by alvis** [reply]
I want it too. Every time I’m checking who and when a change was made, I have to switch from the app to the web page. It’s annoying.
